collector/deploy/kindling-collector-config.yml

```diff
@@ -79,14 +79,40 @@ processors:
     # When using otlp-grpc / stdout exporter , this option supports to
     # send trace data in the format of ResourceSpan
     need_trace_as_span: false
+  aggregateprocessor:
+    # Aggregation duration window size. The unit is second.
+    ticker_interval: 5
+    aggregate_kind_map:
+      request_total_time:
+        - kind: sum
+        - kind: avg
+          output_name: request_total_time_avg
+        - kind: count
```

Comment:
we also need a `last` value for request_total_time for trace as metric

Reply:
There is no `traceAsmetric` generated when the `aggregateprocessor` consumes data. I'd like to work around this problem later.